### PR TITLE
Fix first version + cleaning

### PR DESCRIPTION
**Pull request by lapineige**

### CODE REVIEW

**Comment by lapineige** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -66,7 +66,7 @@ ynh_script_progression --message="Registering the runner on Peertube server..."
 
 pushd "$install_dir"
   ynh_use_nodejs
-  ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production $ynh_node peertube-runner register --url $peertube_url --registration-token $runner_token --runner-name $app\_$(hostname)
+  ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production $ynh_node ./nodes_modules/.bin/peertube-runner register --url $peertube_url --registration-token $runner_token --runner-name $app\_$(hostname)
```
Comment (Tagadda) [inline scripts/install]:
```suggestion
  ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production npx peertube-runner register --url $peertube_url --registration-token $runner_token --runner-name $app\_$(hostname)
```

File: conf/systemd.service [hunk 2]
```diff
@@ -9,7 +9,7 @@ Environment=NODE_CONFIG_DIR=__INSTALL_DIR__/config
 Environment="__YNH_NODE_LOAD_PATH__"
 User=__APP__
 Group=__APP__
-ExecStart=__YNH_NODE__ __INSTALL_DIR__/peertube-runner server
+ExecStart=__YNH_NODE__ __INSTALL_DIR__/node_modules/@peertube/peertube-runner server
```
Comment (Tagadda) [inline conf/systemd.service:12]:
```suggestion
ExecStart=npx peertube-runner server
```

File: scripts/install [hunk 3]
```diff
@@ -62,8 +62,14 @@ ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$ap
...
 # RUNNER REGISTRATION
 #=================================================
+ynh_script_progression --message="Registering the runner on Peertube server..." --weight=1
 
-peertube-runner register --url runner_url --registration-token runner_token --runner-name $app$(hostname)
+pushd "$install_dir"
+  ynh_use_nodejs
+  #ynh_exec_warn ynh_exec_as  $app echo $(ls -a "$install_dir/node_modules/@peertube")
+  ynh_exec_warn ynh_exec_as  $app echo $(systemctl status peertube_remote_runner)
+  ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production $ynh_node "$install_dir/node_modules/@peertube/peertube-runner" register --url $peertube_url --registration-token $runner_token --runner-name $app\_$(hostname)
```
Comment (Tagadda) [inline scripts/install:66]:
```suggestion
  ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production npx peertube-runner register --url $peertube_url --registration-token $runner_token --runner-name $app\_$(hostname)
```

File: conf/systemd.service (hunk 2 above)
Comment (lapineige) [reply conf/systemd.service:12]: What is the difference ?

File: scripts/install (hunk 3 above)
Comment (lapineige) [reply scripts/install:66]: (same)

**Comment by lapineige** [comment]
Huge big up to @Chocobozzz who helped me to make this actually work !! 🎉 ❤️

**Comment by lapineige** [comment]
!testme

**Pull request merged**